logstash-core-event/spec/logstash/event_spec.rb

```diff
@@ -82,12 +82,32 @@
         expect(event["reference_test"]).not_to eq(data)
       end
 
-      it "should not return a Fixnum reference" do
+      it "should return a Fixnum" do
```

Comment:
Let's put these in a context for things that should only work in 2.3 and older as a way of documenting for ourselves that these should be removed (and also fail) on 5.0